deploy-manage/remote-clusters/_snippets/eck_rcs_connect_intro.md

```diff
@@ -1,5 +1,15 @@
-On the local deployment, add the remote ECK cluster using {{kib}} or the {{es}} API with the following connection settings:
+<!--
+This snippet is in use in the following locations:
+- eck-remote-clusters-from-external.md
+- ec-enable-ccs-for-eck.md
+- ece-enable-ccs-for-eck.md
 
-* **Remote address**: Use the FQDN or IP address of the LoadBalancer service, or similar resource, you created to expose the remote cluster server interface (for API key-based authentication) or the transport interface (for TLS certificate-based authentication).
+It requires local_type_generic substitution to be defined
+-->
+On the local {{local_type_generic}}, add the remote ECK cluster using {{kib}} or the {{es}} API with the following connection settings:
+
+* **Remote address**: Use the FQDN or IP address of the LoadBalancer service, or similar resource, you created to expose the remote cluster server interface (for API key-based authentication) or the transport interface (for TLS certificate-based authentication), and the port.
```

Comment:
this is quite dense. I suggest some child bullets so it's easier to scan. something like ... 

```suggestion
* **Remote address**: Use the FQDN or IP address of the LoadBalancer service or alternative resource that you created to expose the remote cluster.

	* For API key-based authentication, use the server interface address and port.
	* For TLS certificate-based authentication, use the transport interface address and port.
```